webapp/components/DeleteData/DeleteData.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <base-card class="delete-data">
+  <base-card v-if="currentUser.role !== 'admin'" class="delete-data">
```

Comment:
I don't think we have agreed on this, therefore I would open an issue and we can discuss it, not just make this decision alone.

Reply:
so i changed it